### PR TITLE
Add Tutorial(1. index postgres; 2. setup superset preview client) for the doc

### PR DESCRIPTION
**Pull request by feng-tao**
### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [ ] PR title addresses the issue accurately and concisely.
- [ ] PR includes a summary of changes.
- [ ] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: docs/tutorials/data-preview-with-superset.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# How to setup a preview client with Apache Superset
+
+In the previous [tutorial](docs/tutorials/index-postgres.md), we talks about how to index the table metadata
```
Comment (ttannis) [inline docs/tutorials/data-preview-with-superset.md:3]:
```suggestion
In the previous [tutorial](docs/tutorials/index-postgres.md), we talked about how to index the table metadata
```

File: docs/tutorials/data-preview-with-superset.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+for a postgres database. In this tutorial, we will walk through how to config data preview for this `films` table
```
Comment (ttannis) [inline docs/tutorials/data-preview-with-superset.md:4]:
```suggestion
for a postgres database. In this tutorial, we will walk through how to configure data preview for this `films` table
```

File: docs/tutorials/data-preview-with-superset.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+   
+2. We need to add the postgres database to superset as the following:
+![](../img/tutorials/superset-add-db.png)
+
+3. We could verify the content of the `films` table using superset's sqlab feature:
+![](../img/tutorials/superset-sqllab-verify.png)
+
+4. Next, We need to build a preview client following this [guide](../frontend/docs/examples/superset_preview_client.md) 
+and the [example client code](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/base/examples/example_superset_preview_client.py).
+There are couples of things keep in mind:
```
Comment (ttannis) [inline docs/tutorials/data-preview-with-superset.md:46]:
```suggestion
There are a couple of things keep in mind:
```

File: docs/tutorials/data-preview-with-superset.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    - We could start with an unauthenticated Superset([example superset config](https://gist.github.com/feng-tao/b89e6faf7236372cef70a44f13615c39)), 
+    but in production, we will need to send the impersonate info to Superset
```
Comment (ttannis) [inline docs/tutorials/data-preview-with-superset.md:48]: So the impersonation support is in open source Superset as well?

File: docs/tutorials/data-preview-with-superset.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    to properly verify whether the given user could view the data.
+    - When we build the client, we could need configure the database id instead of database name when send the request to superset.
+  
+5. Once we configure the preview client, put it in the frontend service entry point and restart the frontend.
```
Comment (ttannis) [inline docs/tutorials/data-preview-with-superset.md:52]: Can link here for examples of using the entry point. https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/configuration.md#python-entry-points

File: docs/tutorials/data-preview-with-superset.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+3. We could verify the content of the `films` table using superset's sqlab feature:
+![](../img/tutorials/superset-sqllab-verify.png)
+
+4. Next, We need to build a preview client following this [guide](../frontend/docs/examples/superset_preview_client.md) 
+and the [example client code](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/base/examples/example_superset_preview_client.py).
+There are couples of things keep in mind:
+    - We could start with an unauthenticated Superset([example superset config](https://gist.github.com/feng-tao/b89e6faf7236372cef70a44f13615c39)), 
+    but in production, we will need to send the impersonate info to Superset
+    to properly verify whether the given user could view the data.
+    - When we build the client, we could need configure the database id instead of database name when send the request to superset.
```
Comment (ttannis) [inline docs/tutorials/data-preview-with-superset.md:50]: A little confused by this statement -- in the open source version does it work if you send the database name in the request?

File: docs/tutorials/data-preview-with-superset.md (hunk 3 above)
Comment (jornh) [reply docs/tutorials/data-preview-with-superset.md:46]: Even: things **to** keep in mind

File: docs/tutorials/data-preview-with-superset.md (hunk 4 above)
Comment (feng-tao) [reply docs/tutorials/data-preview-with-superset.md:48]: i check which Lyft superset override that endpoint to do personal authorization with TOM, while OSS superset does it a similar but rely on FAB to do impersonation API security check.

File: docs/tutorials/data-preview-with-superset.md (hunk 6 above)
Comment (feng-tao) [reply docs/tutorials/data-preview-with-superset.md:50]: still requires db id, let me clarify a bit.

**Comment by feng-tao** [comment]
@jornh @ttannis update the pr per comment.

**Pull request merged**